### PR TITLE
Low health vignette

### PR DESCRIPTION
**Pull request by FluxCapacitor2**
Renders a vignette, just like when the player is outside of the world border, when the player's health is low. It gets more opaque as the player's health decreases. Does not conflict with the world border fix ("Red screen fix").

Resolves suggestion 2347.

### CODE REVIEW

File: src/main/kotlin/gg/skytils/skytilsmod/utils/RenderUtil.kt [hunk 1]
```diff
@@ -758,6 +758,43 @@ object RenderUtil {
...
+        UGraphics.color4f(
+            (1f - (color.red / 255f)) * (color.alpha / 255f),
+            (1f - (color.green / 255f)) * (color.alpha / 255f),
+            (1f - (color.blue / 255f)) * (color.alpha / 255f),
+            1f
+        )
+
+        val sr = UResolution
+
+        UGraphics.bindTexture(0, vignetteTexPath)
```
Comment (FluxCapacitor2) [inline src/main/kotlin/gg/skytils/skytilsmod/utils/RenderUtil.kt:787]: To be honest I have no idea what the first parameter (`index`) actually does, but setting it to 0 has not caused any problems

File: src/main/kotlin/gg/skytils/skytilsmod/features/impl/misc/MiscFeatures.kt [hunk 2]
```diff
@@ -149,7 +150,7 @@ class MiscFeatures {
                 }
             }
             if (unformatted == "Your zapper is temporarily fatigued!") {
-                val duration = Duration.milliseconds(blockZapperCooldownExpiration - System.currentTimeMillis())
+                val duration = (blockZapperCooldownExpiration - System.currentTimeMillis()).milliseconds
```
Comment (My-Name-Is-Jeff) [inline src/main/kotlin/gg/skytils/skytilsmod/features/impl/misc/MiscFeatures.kt]: This won't work cause essential

File: src/main/kotlin/gg/skytils/skytilsmod/utils/RenderUtil.kt (hunk 1 above)
Comment (My-Name-Is-Jeff) [reply src/main/kotlin/gg/skytils/skytilsmod/utils/RenderUtil.kt:787]: I don't know if this method actually exists or not

**Review by My-Name-Is-Jeff — CHANGES_REQUESTED** [review]

File: src/main/kotlin/gg/skytils/skytilsmod/features/impl/misc/MiscFeatures.kt (hunk 2 above)
Comment (FluxCapacitor2) [reply src/main/kotlin/gg/skytils/skytilsmod/features/impl/misc/MiscFeatures.kt]: IntelliJ auto corrected it without telling me :smile:

**Pull request merged**